function.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Rename file to `functions.php` because it holds more than one function

Reply:
Filename has been changed